### PR TITLE
Add content archive tables to DS content page

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7224/app_12409_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7225/app_12409_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7229/app_12409_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7229/HighFidelity-Beta-PR12409-676ae12455db7340cafcb44145382962e834e7d5.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7229/HighFidelity-Beta-PR12409-676ae12455db7340cafcb44145382962e834e7d5.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7254/app_12409_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7254/HighFidelity-Beta-PR12409-68ecefb27b641958fe0aa08be9c8d6f0f565e8b0.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7254/HighFidelity-Beta-PR12409-68ecefb27b641958fe0aa08be9c8d6f0f565e8b0.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7259/app_12409_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7259/HighFidelity-Beta-PR12409-4687b24d0dc0c0f9a8f64c15074f101ccb3f3222.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7259/HighFidelity-Beta-PR12409-4687b24d0dc0c0f9a8f64c15074f101ccb3f3222.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7263/app_12409_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7263/HighFidelity-Beta-PR12409-74d5d6cb17febb3b73d963e36302e585e129564f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7263/HighFidelity-Beta-PR12409-74d5d6cb17febb3b73d963e36302e585e129564f.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7266/app_12409_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7266/HighFidelity-Beta-PR12409-e12a08ba6785fc5f6bfcc9d3733b8f2201d182b8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7266/HighFidelity-Beta-PR12409-e12a08ba6785fc5f6bfcc9d3733b8f2201d182b8.exe

File: domain-server/src/DomainServer.cpp [hunk 1]
```diff
@@ -2244,17 +2251,52 @@ bool DomainServer::handleHTTPRequest(HTTPConnection* connection, const QUrl& url
...
             QList<FormData> formData = connection->parseFormData();
 
             if (formData.size() > 0 && formData[0].second.size() > 0) {
-                // invoke our method to hand the new octree file off to the octree server
-                QMetaObject::invokeMethod(this, "handleOctreeFileReplacement",
-                                          Qt::QueuedConnection, Q_ARG(QByteArray, formData[0].second));
+                auto& firstFormData = formData[0];
+
+                // check the file extension to see what kind of file this is
+                // to match sure we handle this filetype for a content restore
```
Comment (Atlante45) [inline domain-server/src/DomainServer.cpp]: "to make sure"?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7284/app_12409_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7285/app_12409_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7285/HighFidelity-Beta-PR12409-de75fe8e9f04618b0e3e9febbc6c2c6481d37e18.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7285/HighFidelity-Beta-PR12409-de75fe8e9f04618b0e3e9febbc6c2c6481d37e18.exe

**Pull request merged**